### DISCUSSION TITLE
Kong hang was when adding target from Upstream.

### DISCUSSION THREAD

**Discussion by Joou** [Kong/kong#9482] [Help]
Kong hang was when adding target from Upstream. We restarted Kong and port 8080 was not opened from localhost or outside. We checked the Kong Access log, error log, Postgresql log and server logs. We couldn't find any errors. The database is on the same server. The server is virtual. Has anyone experienced this after adding targets from upstream? We couldn't find the error in any logs.

Kong  version 2.0.1
Database : Postgresql 11
web interface : Konga 0.14.7

Thanks.

**Comment by samugi** [comment]
Hello @Joou, can you replicate this on the latest Kong version: 3.0.0? If so, could you describe specific steps to reproduce?
Thank you.
